managed/src/main/java/com/yugabyte/yw/controllers/TabletServerController.java

```diff
@@ -86,11 +87,10 @@ public Result listTabletServers(UUID customerUUID, UUID universeUUID) {
       final int masterHttpPort = universe.getUniverseDetails().communicationPorts.masterHttpPort;
       final String masterLeaderUrl =
           String.format("http://%s:%s/api/v1/tablet-servers", masterLeaderIPAddr, masterHttpPort);
-
       response = apiHelper.getRequest(masterLeaderUrl);
     } catch (Exception e) {
```

Comment:
Can we remove this?

Reply:
Maybe. But not sure if the error will be clear enough in that case. Lets keep it.

Reply:
There is a log too and I think we need the log so keeping it.